src/mcts.rs

```diff
@@ -288,6 +331,16 @@ impl<'a> Searcher<'a> {
         (pv, score)
     }
 
+    fn get_best_move(&self) -> Move {
+        let idx = self.tree.get_best_child(self.tree.root_node());
+        let action = self.tree.edge(self.tree.root_node(), idx);
+        Move::from(action.mov())
```

Comment:
This is unused, can you either use it or remove it?

Reply:
It's used to update the best_move_changes variable: 
```rust
let new_best_move = self.get_best_move();
if new_best_move != best_move {
    best_move = new_best_move;
    best_move_changes += 1;
}
```

Reply:
@XInTheDark ah, I see, in that case can you also use it at the end of the search function?

Reply:
> @XInTheDark ah, I see, in that case can you also use it at the end of the search function?

Apparently the search function needs to return both best move and eval, and get_best_move() only returns best move as a speedup. Although I think we can just do get_pv(0) instead?

Reply:
Nah no worries then